cub/benchmarks/bench/scan/applications/P1/log-cdf-from-log-pdfs.cu

```diff
@@ -0,0 +1,188 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
nit: if `P1` is not functionally needed for the build system, I am against making paths over-complicated. Under   `bench/scan` we have:

```
bench/scan
|
|---applications
|   |---P1
|       |---benches
|
|---exclusive
    |---benches
```

I am fine adding `/applications` (though hard for outsiders to track that would rather look into `examples` most probably) but `/P1` feels redundant.